pkg/capabilities/v2/protoc/pkg/test_capabilities/importclash/clash_sdk_gen.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/protoc, DO NOT EDIT.
+
+package importclash
+
+import (
+	"errors"
+
+	"google.golang.org/protobuf/types/known/anypb"
+
+	pb3 "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/protoc/pkg/test_capabilities/importclash/p1/pb"
+	pb4 "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/protoc/pkg/test_capabilities/importclash/p2/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2"
+	sdkpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb"
```

Comment:
how is `sdkpb` a known alias?

Reply:
it's not a generated alias, I added that one a while ago.